src/utils/validations.ts

```diff
@@ -2,7 +2,7 @@ export const validateLoginFields = (
   email?: string,
   password?: string,
 ): boolean => {
-  return !!(email && email.includes('@') && password && password.length > 8)
+  return !!(email && email.includes('@') || password && password.length > 8)
```

Comment:
created a bug here so it enables the `Log in` button with a valid email or a valid password (it should be both)